test/04LessonFourTest.js

```diff
@@ -4,38 +4,53 @@ var assert = require('assert');
 
 function arrayPop(arr) {
     // should remove the last item in the array, then return the arr
-
+    var poppedItem = arr.pop();
```

Comment:
Technically, you don't need to declare this variable if you never use it.

Same goes for the functions that follow.

Good job, though.